src/Core/Components/DataGrid/Columns/ColumnOptionsLabels.cs

```diff
@@ -10,6 +10,17 @@ public record ColumnOptionsLabels
     /// </summary>
     public string OptionsMenu { get; set; } = "Filter";
 
+    /// <summary>
+    /// Gets or sets the icon to show in the column menu
+    /// </summary>
+    public Icon? Icon { get; set; } = new CoreIcons.Regular.Size16.Filter();
+
+    /// <summary>
+    /// Gets or sets the position the icon can be showed in.
+    /// Possible options are 'start' (default) or 'end'
+    /// </summary>
+    public string Slot { get; set; } = "start";
```

Comment:
Why do you allow other string values? Why not a bool `bool IconPositionStart`?

Reply:
Yeah, can change that.